homeassistant/util/unit_system.py

```diff
@@ -167,6 +167,37 @@ def as_dict(self) -> dict:
             SPEED: self.speed_unit
         }
 
+    def convert(self, state, unit_of_measure) -> Tuple[float, str]:
```

Comment:
Util methods should not be home assistant aware, so you should probably replace `state` with `value`, or similar.